templates/activity.html

```diff
@@ -1,41 +1,29 @@
-<!doctype html>
-<html>
+{% extends 'includes.html' %}
+{% block content %}
+    <div class="section no-pad-bot" id="index-banner">
+        <div class="container">
+            <h1 class="header center  col-test">Welcome</h1>
+            <div class="row center">
+                <h5 class="header col s12 light">Hello, world. You are at the {{ org_name }} community GCI website.</h5>
+            </div>
+        </div>
+        {% include 'issues_graph.html' %}
 
-<head>
-    <title>Community Activity</title>
-    <style>
-    canvas {
-        -moz-user-select: none;
-        -webkit-user-select: none;
-        -ms-user-select: none;
-    }
-    </style>
-    <meta http-equiv="content-type" content="text/html; charset=utf-8" />
-</head>
-
-<body>
-    <h1>Community Activity</h1>
-    <br>
-    <br>
-    <div style="width:75%;">
-        <canvas id="canvas"></canvas>
     </div>
-    <select id="chartType">
-        <option value="Month">Year</option>
-        <option value="Week">Month</option>
-        <option value="Day">Week</option>
-    </select>
-    <br>
-    <br>
+
+    {% include 'footer.html' %}
+    <!--Import jQuery before materialize.js-->
     <script src="https://cdnjs.cloudflare.com/ajax/libs/Chart.js/2.7.1/Chart.bundle.js"></script>
-    <script src="https://ajax.googleapis.com/ajax/libs/jquery/3.2.1/jquery.min.js"></script>
-    <script src="/static/charts.js"></script>
+    <script src="../static/js/charts.js"></script>
     <script>
-    $('#chartType').on('change', function(){
-        updateChart($('#chartType').val())
-    });
-    updateChart($('#chartType').val());
+        $(document).ready(function () {
+            $('select').material_select();
+        });
+        $('#chartType').on('change', function () {
+            updateChart($('#chartType').val())
+        });
+        updateChart($('#chartType').val());
     </script>
+{% endblock %}
 </body>
```

Comment:
please try to have balanced templates, by moving `</body></html>` into the template which opens those tags.